specs/validator/0_beacon-chain-validator.md

```diff
@@ -312,40 +312,40 @@ Set `attestation.aggregate_signature = signed_attestation_data` where `signed_at
 ```python
 attestation_data_and_custody_bit = AttestationDataAndCustodyBit(
     attestation.data,
-    False,
+    0b0,
```

Comment:
Maybe just use 0? 0b0 looks better to an experienced python coder but may give a "wtf m8" reaction to non-pythonistas.